pkg/aws/ec2/pricing_map.go

```diff
@@ -184,27 +238,47 @@ func weightedPriceForInstance(price float64, attributes Attributes) (*Prices, er
 	}, nil
 }
 
-func (spm *StructuredPricingMap) GetPriceForInstanceType(region string, instanceType string) (*Prices, error) {
-	spm.m.RLock()
-	defer spm.m.RUnlock()
-	if _, ok := spm.Regions[region]; !ok {
+func (cpm *ComputePricingMap) GetPriceForInstanceType(region string, instanceType string) (*Prices, error) {
+	cpm.m.RLock()
+	defer cpm.m.RUnlock()
+	if _, ok := cpm.Regions[region]; !ok {
 		return nil, ErrRegionNotFound
 	}
-	price := spm.Regions[region].Family[instanceType]
+	price := cpm.Regions[region].Family[instanceType]
 	if price == nil {
 		return nil, ErrInstanceTypeNotFound
 	}
-	return spm.Regions[region].Family[instanceType], nil
+	return cpm.Regions[region].Family[instanceType], nil
+}
+
+func (spm *StoragePricingMap) GetPriceForVolumeType(region string, volumeType string, size int32) (float64, error) {
```

Comment:
yup, this was somewhat in the back of my mind when I dropped the `/h` from the docs, but completely forgot to kept that going to the actual metric calculation 😅

I'm thinking it's better to divide and get the hourly cost, than go for a monthly metric like they do, right? hours are the same length every time.

Reply:
Went for hourly cost, since all metrics are hour based, and GKE module does hourly metrics for volumes as well.

And yes, the API gives back the prices with the unit `Gi/Mo`, like the docs, so we need to convert from monthly to hourly, and also yes, they use 30 days months in the docs. So you need to do some math one way or another, because they don't bill on 30 day periods.